care/utils/serializer/external_id_field.py

```diff
@@ -0,0 +1,39 @@
+import uuid
+
+from django.core.exceptions import ObjectDoesNotExist
+from rest_framework import serializers
+from rest_framework.fields import empty
+
+
+class UUIDValidator(object):
+    def __call__(self, value):
+        try:
+            return uuid.UUID(value)
+        except ValueError:
+            raise serializers.ValidationError("invalid uuid")
+
+
+class ExternalIdSerializerField(serializers.Field):
+    def __init__(self, queryset=None, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.queryset = queryset
+
+    def get_validators(self):
+        validators = super().get_validators()
+        validators += [UUIDValidator()]
+        return validators
+
+    def to_internal_value(self, data):
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Method could be a function](https://app.codacy.com/gh/coronasafe/care/pullRequest?prid=5498155)